camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/spi/dsl/DslYamlTest.java

```diff
@@ -66,6 +71,63 @@ public void testDslSelf() throws Exception {
     }
 
     @Test
+    public void testDslSingleTag() throws Exception {
```

Comment:
Why are these tests in the `DslYamlTest` class? This class is for testing `$brooklyn:...`.

I'd probably create a new test class for this: `org.apache.brooklyn.camp.brooklyn.TagsYamlTest`.